dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantVectorStoreRecordCollection.cs

```diff
@@ -526,31 +527,6 @@ public virtual async Task<VectorSearchResults<TRecord>> VectorizedSearchAsync<TV
         return new VectorSearchResults<TRecord>(mappedResults.ToAsyncEnumerable());
     }
 
-    /// <summary>
-    /// Resolve the vector field name to use for a search by using the storage name for the field name from options
-    /// if available, and falling back to the first vector field name if not.
-    /// </summary>
-    /// <param name="optionsVectorFieldName">The vector field name provided via options.</param>
-    /// <returns>The resolved vector field name.</returns>
-    /// <exception cref="InvalidOperationException">Thrown if the provided field name is not a valid field name.</exception>
-    private string ResolveVectorFieldName(string? optionsVectorFieldName)
```

Comment:
Great that we're starting to move common functionality into a single place like this. A bit later we can take a general look and probably make VectorStoreRecordPropertyReader into something like a proper VectorStoreModel, which contains all the knowledge and functionality around the modeling of a collection etc.